Source/WebCore/rendering/RenderText.cpp

```diff
@@ -1329,6 +1330,73 @@ UChar RenderText::previousCharacter() const
     return previousString[previousString.length() - 1];
 }
 
+static String convertToFullSizeKana(const String& string)
+{
+    // https://www.w3.org/TR/css-text-3/#small-kana
+    static constexpr std::pair<uint16_t, uint16_t> kanasMap[] = {
```

Comment:
We could use `std::pair<UChar, UChar>` here and avoid having to type cast below.

Reply:
Fixed